builder/build.go

```diff
@@ -105,3 +105,14 @@ func buildFlagString(nocache bool, squash bool, httpProxy string, httpsProxy str
 
 	return buildFlags
 }
+
+func ensureHandlerPath(image string, handler string) error {
+	if _, err := os.Stat(handler); err != nil {
+		fmt.Printf("Unable to build %s, %s is an invalid path\n", image, handler)
```

Comment:
Good changes. Can we have this ensure function actually just return an error instead and then return that in the command so it bubbles up to the main handler?

Reply:
@alexellis I chose to put 2 fmt here to avoid repeating them in `BuildImage()` function, the `ensure` func returns already an error
When we re-work on build command, we can remove these fmt then.

Reply:
In this instance I'd rather have the repetition in the calling code and a simpler function that doesn't print messages and also return an error. It should do one or the other.